### ISSUE TITLE
[CoE Starter Kit - Issue] Admin | Add Maker To Group -> BadRequest

### ISSUE THREAD

**Issue by lucasubieta** [microsoft/coe-starter-kit#9227]
### Does this question already exist in our backlog?

- [x] I have checked and confirm this is a new question.

### What is your question?

Hi there!,

I'm having problems with this flow -> statusCode 400

The user is the owner of the makers group, so I shouldn't have any problems. Any suggestions?

Thanks a lot! 😊

![Image](https://github.com/user-attachments/assets/74080708-d53a-4389-9bc9-6a9f2ce7b8b7)

### What solution are you experiencing the issue with?

None

### What solution version are you using?

July 24

### What app or flow are you having the issue with?

Admin | Add Maker To Group 

### What method are you using to get inventory and telemetry?

None

**Comment by Jenefer-Monroe** [comment]
My guess is that the **_HTTP with Microsoft Entra ID (preauthorized)_** connection is broken or misconfigured.

Please go to connections and make sure the connection for these is as follows (or your tenant equivalent if you are not a commercial tenant: https://graph.microsoft.com/
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/f49797b8-0f48-4884-a026-2580bbc8b96f)
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/a31b22b8-bc81-4d81-a5b7-80dc740f2b82)

Sometimes the product creates one of these connected to the envt and this will not work.

**Comment by lucasubieta** [comment]
Hi Jenefer,
 
Thanks a lot for your reply.

I just reconfigured the 'HTTP with Microsoft Entra ID' and 'Office 365 Groups' connectors. I also checked the Office 365 app registration in Azure (not sure if it’s related, but just in case), and it’s all set but still not working :(


![Image](https://github.com/user-attachments/assets/f262c918-8b9f-4c60-86b4-638b131f0c34)

`I just reconfigured the 'HTTP with Microsoft Entra ID' and 'Office 365 Groups' connectors. I also checked the Office 365 app registration in Azure (not sure if it’s related, but just in case), and it’s all set."

`

**Comment by silvafr1** [comment]
Hello, 
Isn´t because of the new Microsoft change?

https://admin.microsoft.com/AdminPortal/home#/MessageCenter/:/messages/MC918447?MCLinkSource=DigestMail
https://learn.microsoft.com/en-us/connectors/webcontents/#authorize-the-connector-to-act-on-behalf-of-a-signed-in-user

Thank you!

**Comment by Jenefer-Monroe** [comment]
No, that change has been delayed while they continue to iterate on the feature. 
Also, oops I hadn't had my coffee yet :) That is the group connector not the preauth one lol. 

This call uses an environment variable, lets go make sure its correctly configured.
Note that it needs to be the group GUID, not its display name or email. 
![Image](https://github.com/user-attachments/assets/8306368d-5315-4f56-bc9d-d706028adf4b)

**Comment by lucasubieta** [comment]
haha! i really appreaciate your work Jenefer 💯, I checked the group connector as well and it's ok, also reconfigured it. 
Regarding the variable, i would say that it's ok, just doube check the id as well..

![Image](https://github.com/user-attachments/assets/c9928c57-51b6-49a0-bc2d-0082359cae1d)

**Comment by Jenefer-Monroe** [comment]
great can you please restart the flow and resubmit the failure? sometimes flows cache env vars.

**Comment by Jenefer-Monroe** [comment]
also please double check to make sure your connections are healthy 
![Image](https://github.com/user-attachments/assets/b78a935b-fab6-457a-aa78-3690e3c44d89)

**Comment by lucasubieta** [comment]
i checked on the flow run that is taking the correct env var. I also tried to disable and enable it again, still the same

![Image](https://github.com/user-attachments/assets/0508741f-60b8-434d-93f1-984dd58e752e)

**Comment by Jenefer-Monroe** [comment]
What is the specific error message you get from this call? 
![Image](https://github.com/user-attachments/assets/41fd678a-1646-4713-96e2-8cfd6ffbd301)

And can you call this action from a separate flow? We arent doing anything special here, just calling **_List Group Members_**

**Comment by lucasubieta** [comment]
Hi! I just tried a simple flow with the "List group members" action pointing to the same group, and it worked fine.

This is the error I'm getting from the CoE flow. I even replaced the action in the flow with the same action, just in case, but I'm still seeing the same error;
![Image](https://github.com/user-attachments/assets/71d4a71b-a1d1-457f-8a0e-54b7c68609de)

Thanks!,

**Comment by lucasubieta** [comment]
Hi again! I just tried the flow again, replacing the "group variable" with the group itself, and it worked fine. So the issue must be with the "makers group variable." I checked, and it’s configured properly, so I’m not sure what could be causing this.

If I leave it as it is, with the modified flow, I’ll need to remember to change it again whenever I update the solution, right? Since it will be replaced with the original flow.

Thanks

**Comment by Jenefer-Monroe** [comment]
You're right you'll have an unmanaged layer. Lets try to fix it one more time. 
My guess is that it had a space at the beginning or end or something like that.
I suggest you do the following: 

Go to default solution in the CoE Environment > Environment Variables > find this env var > delete the current value > add it again
![Image](https://github.com/user-attachments/assets/98a88970-e572-4a78-be76-f60c60ef38ad)
![Image](https://github.com/user-attachments/assets/af8844fb-7b3e-4b01-8455-20c2cfeeb434)
![Image](https://github.com/user-attachments/assets/d3f568cd-f489-4774-971a-f0c3229224c0)

**Comment by Jenefer-Monroe** [comment]
closing out as no further action for starter kit team
